master-server/src/main/java/com/replicated_log/master_server/controller/rest/MasterRestController.java

```diff
@@ -0,0 +1,80 @@
+package com.replicated_log.master_server.controller.rest;
+
+import com.replicated_log.master_server.model.Ack;
+import com.replicated_log.master_server.model.AckStatusCode;
+import com.replicated_log.master_server.model.Address;
+import com.replicated_log.master_server.model.Item;
+import com.replicated_log.master_server.service.AckService;
+import com.replicated_log.master_server.service.MasterService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(MasterRestController.MASTER_URL)
+@CrossOrigin("*")
+public class MasterRestController {
+
+    private final Logger LOG = LogManager.getLogger(MasterRestController.class);
```

Comment:
just to share - there is lombok lib which is simplify such initialization via @Slf4j annotation

Reply:
added @slf4j annotation for logging